nixos/modules/services/web-servers/unit/default.nix

```diff
@@ -17,6 +17,16 @@ in {
         defaultText = "pkgs.unit";
         description = "Unit package to use.";
       };
+      user = mkOption {
```

Comment:
Many modules do provide a user and group option, though if you don't explicitly need to please consider just hard coding the the user and group and not leave an option.

Reply:
I can not say whether me need a hard coding fixation of the user or not. Need to check working with nginx with rooless mode